PageViews/PageViews.module.less

```diff
@@ -17,11 +17,12 @@
 	}
 
 	.navButtonContainer {
-		overflow: hidden;
-		z-index: 10;
-		top: ~"calc(50% - " (@sand-button-height + 12) / 2 ~")";
 		position: absolute;
 		width: 100%;
+		height: @sand-button-height * 2;
+		top: ~"calc(50% - " (@sand-button-height * 2) / 2 ~")";
```

Comment:
It'd be better if we make `@sand-button-height * 2` as a less variable for the future.